Проблема: давать возможность добавлять дата-атрибуты без значений

### Ошибка из терминала

 36:39  error  The attribute "data-form-validate" requires a value  htmlacademy/attr-req-value

### Часть кода на котором произошла ошибка. Какая конфигурация использовалась
```html
<!DOCTYPE html>
<html lang="ru">

<head>
  <meta charset="UTF-8">
  <title>SoftLine</title>
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <link href="css/styles.css" rel="stylesheet">
</head>

<body>
  <div class="wrapper">
    <header class="header">
      <div class="header__container container">
        <a class="header__logo logo" href="#" aria-label="Логотип SoftLine">
          <img src="#" alt="" width="100" height="100">
        </a>
      </div>
    </header>

    <main class="main">

    </main>
    <footer class="footer">
      <div class="container">
        Футер
      </div>
    </footer>

    <div class="modal modal--preload" data-modal="feedback">
      <div class="modal__wrapper">
        <div class="modal__overlay" data-close-modal=" "></div>
        <div class="modal__content">
          <p class="modal__title">Оставьте заявку</p>

          <div class="modal__form form" data-form-validate data-callback="base">

          </div>
          <button class="modal__close-btn btn-reset" type="button" aria-label="Закрыть попап" data-close-modal=" ">
            <svg width="15" height="15">
              <use href="img/sprite.svg#icon-close"></use>
            </svg>
          </button>
        </div>
      </div>
    </div>
  </div>

</body>

</html>
```
### Описание проблемы

Сейчас правило ожидает значения для дата-атрибута `data-modal="feedback"`, но нужно дать возможность добавлять в разметку атрибуты без значения, например `data-form-validate`

--- a/package.json
+++ b/package.json
@@ -19,6 +19,6 @@
   "homepage": "https://github.com/htmlacademy/linthtml-config-htmlacademy#readme",
   "dependencies": {
     "@linthtml/linthtml": "0.9.5",
-    "linthtml-rules-htmlacademy": "1.0.9"
+    "linthtml-rules-htmlacademy": "1.0.10"
   }
 }

